JobConfig/beam/prolog.fcl

```diff
@@ -5,301 +5,301 @@ BEGIN_PROLOG
-
-         // Make sure the charged particle beam reaches our DS2Vacuum stopping volume
-         {type: notInVolume pars:[TS2Vacuum, TS3Vacuum, TS4Vacuum, TS5Vacuum, Coll31, Coll32, Coll51, TS2InnerCryoShell, TS3InnerCryoShell, TS4InnerCryoShell, TS2CryoInsVac, TS3CryoInsVac, VirtualDetector_Coll31_In, VirtualDetector_Coll32_In, VirtualDetector_Coll31_Out, VirtualDetector_Coll32_Out, Coll31OutRecord, Coll32InRecord, Coll31OutRecord, VirtualDetector_TS4_Bend, VirtualDetector_Coll5_In, VirtualDetector_Coll5_Out, VirtualDetector_Coll5_OutSurf]},
-
-         // This union splits the output of the above cuts
-         // into two exclusive streams; Beam (charged particles entering the DS) and Neutrals ((mostly)neutral particles approaching the CRV
-         { type: union pars:  [
-               {
-                  type: intersection
-                  pars: [
-                     { type: inVolume pars: [DS2Vacuum] },
-                     { type: isCharged }
-                  ]
-                  write: Beam
-               },
-               {  type: constant
-                  value: true
-                  write: Neutrals
-               }
-            ]
-         }
-      ]
-   }
-
-   # define extinction monitor region
-   extMonRegionCuts: {
-      type: intersection
-      pars: [
-         {
-            type: plane
-            normal: @local::mu2eg4BeamDumpNormal
-            // Dump core center: (661.90,  527.40, -19428.2)
-            // A plane 950 mm beyond the core center as measured along
-            // the "beam" direction normal to the dump face.
-            // DeltaZ = L/cos(13.72 degrees)
-            point: [ 661.90, 527.40, -20406.10 ]
-         },
-
-         // Split the output of the "plane" cut above into two exclusive streams
-         { type: union pars:  [
-
-               {  type: intersection
-                  pars: [
-                     {
-                        type: inVolume
-                        pars:
-                        [  collimator1Channel,
-                           HallAir,
-                           ExtMonFNALfilterMagnetAperture,
-                           ExtMonFNALfilterMagnetApertureMarginUp,
-                           ExtMonFNALfilterMagnetApertureMarginDn
-                        ]
-                     },
-                     { type: isCharged }
-                  ]
-                  write: extmonbeam
-               },
-
-               { type: constant  value: true       write: extmonregion }
-            ]
-         }
-      ]
-   }
-#
-# define products that need to be kept
-#
+    ]
+  }
+  #
+  # define products that need to be kept
+  #
   SimKeptProducts : [
-            "keep art::EventIDs_*_*_*",
-	    "keep mu2e::GenParticles_*_*_*",
-            "keep mu2e::GenEventCount_*_*_*",
-            "keep mu2e::StatusG4_*_*_*" 
+    "keep art::EventIDs_*_*_*",
+    "keep mu2e::GenParticles_beamResampler_*_*",
```

Comment:
this is expected right? we want to keep only the beamResampler GenParticles?

Reply:
Good catch.  This was intended to eliminate a duplicate copy of GenParticle that showed up after resampling (pre/post compression) but I see now that it would break the POT output.    I've reverted it.

Reply:
I think this still hasn't been reverted?